src/gluonts/model/simple_feedforward/_network.py

```diff
@@ -120,6 +120,32 @@ def get_distr(self, F, past_target: Tensor) -> Distribution:
         return self.distr_output.distribution(
             distr_args, scale=target_scale.expand_dims(axis=1)
         )
+    def get_distr_arg(self, F, past_target: Tensor):
```

Comment:
```suggestion
    def get_distr_args(self, F, past_target: Tensor):
```

Reply:
Also, the function above (`get_distr`) is now unused? If so, it should be removed since it's redundant